### DISCUSSION TITLE
Ma... è uscito Joypad?

### DISCUSSION THREAD

**Discussion by pdonadeo** [pdonadeo/e-uscito-joypad#30] [General]
Stamattina è uscito su Spotify, mi è arrivata la notifica. Più tardi mi è arrivata anche la notifica del post di Zampa su Instagram ma sul Post non c'è traccia dell'episodio. Gombloddo ai danni di @zampale?

**Comment by zampale** [comment]
Sul post hanno messo l’articolo ma non l’hanno caricato sull’app. Il lancio social è solitamente in un secondo momento. Che pazienza
Il 25 giu 2022, 17:26 +0100, Paolo Donadeo ***@***.***>, ha scritto:

**Reply by agofa** [reply]
Eh sì direi proprio gombloddo!11!

**Reply by pdonadeo** [reply]
Comunque ancora niente…

**Comment by pdonadeo** [comment]
Non ci potete credere…

Sul Post hanno finalmente pubblicato Joypad (sito e app) ma la nostra pagina non si aggiorna, dice ancora che non è uscito niente.

Dopo molto debugging e molto sudore ho realizzato che non è un bug: Il Post ha davanti al sito una cache _per IP_ per cui... dal server su cui è ospitato Joypad la [pagina che viene scaricata per controllare se ci sono nuove puntate](https://www.ilpost.it/podcasts/joypad/) è ancora quella vecchia, in cui si vede solo l'episodio 50 e non il 51.

Quindi riformulo: non è un bug **nostro**, è un bug del Post 😉

**Comment by pdonadeo** [comment]
La cache si è sbloccata mentre scrivevo: adesso abbiamo tutto aggiornato.

**Comment by pdonadeo** [comment]
Questo mese davvero non ci siamo, bisogna sgridare la redazione!

![image](https://user-images.githubusercontent.com/242049/175818465-0cad6152-3ce7-4e67-be49-355ec34b7440.png)

Ne abbiamo saltata una? 😂

**Reply by agofa** [reply]
Sono d'accordo con @pdonadeo, è un evidente complotto contro @zampale, iniziato con la subdola manfrina puntata vs episodio.
Io mi guarderei le spalle, ma noi saremo sempre al suo fianco, un piccolo passo indietro, ma al suo fianco. A meno che gli altri non paghino di più...

**Reply by MarcoJul** [reply]
Mi sono perso questa discussione, mea culpa, ero troppo impegnato a risolvere i MIEI bug😅 

Se però avete bisogno, avete la mia spada, la mia ascia, la mia tastiera. Vogliono depistarci!
